app/controllers/placeholder_controller.rb

```diff
@@ -21,25 +21,16 @@ def home
       redirect_to landing_path
     end
     @showQuizWarning = true
-    @organizations = [
-      '(Mindful) at Berkeley', 
-      '*African American Student Development*', 
-      '*Berkeley Art Studio*', '*LEAD Center*', 
-      '*Native American Student Development*',
-      '(Mindful) at Berkeley', 
-      '*African American Student Development*', 
-      '*Berkeley Art Studio*', '*LEAD Center*', 
-      '*Native American Student Development*',
-      '(Mindful) at Berkeley', 
-      '*African American Student Development*', 
-      '*Berkeley Art Studio*', '*LEAD Center*', 
-      '*Native American Student Development*',
-    ]
+    @organizations = Organization.get_organizations(12, nil, nil)
   end
 
   def generate_orgs

```

Comment:
Coverage for this controller is low because this method is not covered. Should make a test for this. Also should remove the puts to improve test coverage!

Reply:
Correct me if I'm wrong, but you have developed an alternative that does not involve the generate_orgs route at all, so this will be overwritten anyway and I won't bother making any changes for it here.

Reply:
I think we'll still use this controller method so just add the test for this and we'll be fine to merge !

Reply:
Added.